app/_hub/kong-inc/rate-limiting/_index.md

```diff
@@ -132,7 +132,7 @@ params:
       datatype: string
       referenceable: true
       description: |
-        When using the `redis` policy, this property specifies the username to connect to the Redis server when ACL authentication is desired.
+        When using the `redis` policy, this property specifies the username to connect to the Redis server when ACL authentication is desired. This requires Redis v6.0.0+. Note that you can not set it as *default*.
```

Comment:
```suggestion
        When using the `redis` policy, this property specifies the username to connect to the Redis server when ACL authentication is desired. This requires Redis v6.0.0+. This can not be set to **default**.
```